installer/install.bat

```diff
@@ -63,7 +68,8 @@ set PATH=%INSTALL_ENV_DIR%\Library\bin;%PATH%
 
 @rem Download/unpack/clean up InvokeAI release sourceball
 set err_msg=----- InvokeAI source download failed -----
-curl -L %RELEASE_URL%/%RELEASE_SOURCEBALL% --output InvokeAI.tgz
+echo Trying to download "%RELEASE_URL%%RELEASE_SOURCEBALL%"
+curl -L %RELEASE_URL%%RELEASE_SOURCEBALL% --output InvokeAI.tgz
```

Comment:
i think these curl commands should do `--fail-with-body`:
https://superuser.com/questions/590099/can-i-make-curl-fail-with-an-exitcode-different-than-0-if-the-http-status-code-i/1626376#1626376

as-is a 404 produces a `0` exit code, which leads to the weird situation of having InvokeAI.tgz be a text file containing the site's 404 html page.

Reply:
--fail-with-body does not seem to be supported on the curl that I have installed on two of my machines. Rather than try to dig into this right now, I'm going to go ahead and merge without this change.

Reply:
I agree with @lstein here - as well as this works, it's really just past the MVP stage - there is a *ton* of work which could be done, esp. trapping, hardening, retries...